packages/saved-scripts/src/saved-scripts-list-item.tsx

```diff
@@ -69,23 +70,16 @@ function SavedScriptsListItem({
         />
       ) : (
         <SavedScriptsListItemDisplayName
-          className='saved-scripts-list-item__display-name'
-          onClick={() => !isEditing && onSelectScript(script)}
+          className="saved-scripts-list-item__display-name"
+          onClick={() => (isProjectFiles || !isEditing) && onSelectScript(script)}
         >
-          {connectDragSource!(<span>
-            {displayName}
-          </span>)}</SavedScriptsListItemDisplayName>
+          {connectDragSource!(<span>{displayName}</span>)}
+        </SavedScriptsListItemDisplayName>
       )}
-      <SavedScriptsButtonWrapper className='saved-scripts__button-wrapper'>
-        {isStatic || isEditing ? null : (
-          <SavedScriptsEditButton onEdit={() => setIsEditing(!isEditing)}/>
-        )}
-        {isStatic || !isEditing ? null : (
-          <SavedScriptsRemoveButton onRemove={() => onRemoveScript(script)}/>
-        )}
-        {script.isSuggestion || isEditing ? null : (
-          <SavedScriptsExecButton onExec={() => onExecScript(script)}/>
-        )}
+      <SavedScriptsButtonWrapper className="saved-scripts__button-wrapper">
+        {isStatic || isEditing ? null : <SavedScriptsEditButton onEdit={() => setIsEditing(!isEditing)} />}
```

Comment:
I think these ternaries are really hard to read, perhaps we could simplify them somehow?

Reply:
yeah might be something to look at later, I agree its not the easiest to read.